File: server/src/main/java/com/uiptv/model/ThemeCssOverride.java
package com.uiptv.model;

import com.uiptv.shared.BaseJson;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = false)
public class ThemeCssOverride extends BaseJson {
    private String dbId;
    private String lightThemeCssName;
    private String lightThemeCssContent;
    private String darkThemeCssName;
    private String darkThemeCssContent;
    private String updatedAt;
}

File: server/src/main/kotlin/com/uiptv/model/ThemeCssOverride.kt
package com.uiptv.model

import com.uiptv.shared.BaseJson

data class ThemeCssOverride @JvmOverloads constructor(
    var dbId: String? = null,
    var lightThemeCssName: String? = null,
    var lightThemeCssContent: String? = null,
    var darkThemeCssName: String? = null,
    var darkThemeCssContent: String? = null,
    var updatedAt: String? = null
) : BaseJson()
